app/views/users/index.html.erb

```diff
@@ -1 +1,13 @@
-<div>All the Users page</div>
\ No newline at end of file
+<section class="users_section">
+  <% @users.each do |user| %>
+    <%= link_to user_path(user) do %>
+      <div class="user_card">
+        <img src="<%=user.photo%>" alt="<%=user.name%>'s photo"/>
+        <div class="user_data">
+          <h2><%=user.name%></h2>
+          <div class="user_posts_num">Number of posts: <%=user.post_counter%></div>
```

Comment:
[OPTIONAL ]
- Great job implemeting the rendering of all `users` from the `index action` using `@users.each`. A better approach will be to make use of `partials`. I will suggest a code snippet below that does that and you can consider it.
```
 _app/views/users/_user.html.erb_

       

HTML Code Here displaying
        user.photo, user.name and user.posts_counter


 _app/views/users/index.html.erb_

    <%= render @users %>
```